resources/js/Components/Tables/BookingsTable.vue

```diff
@@ -34,17 +56,17 @@
                 Edit
               </button>
             </a>
-              <a v-else-if="booking.status == 'review'" style="cursor:pointer;">
+              <a v-else-if="booking.status == 'review'" :href="'/bookings/'+booking.id+'/view'" style="cursor:pointer;">
                 <button aria-disabled="true" class="h-10 px-5 m-2 text-gray-100 transition-colors duration-150 bg-gray-500 rounded-lg focus:shadow-outline ">
                   In Review
                 </button>
               </a>
-              <a v-else-if="booking.status == 'refused'" >
+              <a v-else-if="booking.status == 'refused'" :href="'/bookings/'+booking.id+'/view'" style="cursor:pointer;">
                 <button aria-disabled="true" class="h-10 px-5 m-2 text-gray-100 transition-colors duration-150 bg-red-900 rounded-lg focus:shadow-outline">
                   Refused
                 </button>
               </a>
-              <a v-else-if="booking.status == 'approved'" >
+              <a v-else-if="booking.status == 'approved'" :href="'/bookings/'+booking.id+'/view'" style="cursor:pointer;">
```

Comment:
might be cleaner to do the v-if else-if logic on the button and not repeat the <a :href> tag 3 times

Reply:
what is the difference? Each button has it's own class anyway. it will still be repeated 3-4 times. At least this way if we need to change anything about each link, we are able to.

Reply:
like if in the future, denied bookings can maybe redirect back to an edit page so that the user can try again.

Reply:
Right now you're repeating the href navigation path 3 times as well as the buttons. If you refactor you'll only be repeating the thing that is actually different aka "button with different color and text"

Reply:
applied changes